docs/developers.md

```diff
@@ -20,9 +20,8 @@
 
 ## Requirements ##
 
-`Logisim-evolution` is written in Java 11, so to build it from sources you need JDK 11
-(or equivalent, e.g. [OpenJDK](https://adoptopenjdk.net/)). To build
-platform specific packages, JDK 14+ is required. We recommend using JDK 16 for your development work with `Logisim-evolution`.
+`Logisim-evolution` is written in Java 14, so to build it from sources you need JDK 14
```

Comment:
How about: "`Logisim-evolution` is written in Java 14. To build it from sources, you need to install [OpenJDK](https://adoptopenjdk.net/) (or equivalent) version 14 or newer."?

Reply:
Fixed